tests/python/unittest/test_operator.py

```diff
@@ -1617,7 +1616,7 @@ def test_convolution_grouping():
         exe2.backward(exe2.outputs[0])
```

Comment:
dtype of arr1 is float64 and dtype of arr2 is float32. Probably a downcast is happening when you do arr2[:] = arr1. Have you tried to fix this and check for flakiness.

Reply:
arr1 and arr2 are both float32. Downcast is happening during populating arr1 by random normal samples. I have made the casting explicit just to avoid errors.